lib/ws-browser.js

```diff
@@ -36,15 +94,51 @@ class Transport extends EventEmitter {
     return this.socket;
   }
 
+  _sendBufferedMessages() {
+    let bufferedAmount = this.socket.bufferedAmount;
+    let toSend = '';
+    while (this._sendBuffer.length !== 0) {
+      const message = this._sendBuffer[0];
+      const utf8messageLength = utf8length(message);
+      if (utf8messageLength + bufferedAmount <= BUFFER_SIZE_LIMIT) {
+        toSend += message;
+        bufferedAmount += utf8messageLength;
+        this._sendBuffer.shift();
+      } else {
+        const chunkLength = BUFFER_SIZE_LIMIT - bufferedAmount;
+        if (chunkLength < 0) {
+          this.socket.close();
+          throw new Error('WebSocket buffer is overflown');
+        }
+        if (chunkLength !== 0) {
```

Comment:
As I mentioned before I'd prefer for this to be `> 0` check.

Reply:
There is no need to make a `> 0` check since I check if the value is `< 0` and then exit before this line in that case, and because of that, checking for `!== 0` here covers all of the other values which are `> 0`.